.flake8

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length=120
```

Comment:
Almost all of the codebase is already conforming to a 80 char max line length (inherited from checks when in OpenStack).  Even though 80 can be short at times, I think it would be best to keep the pattern. Thanks

Reply:
Reverting this check.  119 is the limit for Github PR's, will stay with OpenStack convention.

Reply:
Wondering if line too long warning should be ignored individually for those single-line docstring comments, or should the docstring be split up?

```plaintext
bandit\formatters\screen.py:98:80: E501 line too long (99 > 79 characters)
    """Output issue str returns a list of lines that should be added to the existing lines list."""
                                                                               ^
bandit\formatters\text.py:44:80: E501 line too long (103 > 79 characters)
    """Get verbose details including files in scope, score, SEVERITY and CONFLUENCE, files excluded."""
                                                                               ^
bandit\formatters\text.py:71:80: E501 line too long (99 > 79 characters)
    """Output issue str returns a list of lines that should be added to the existing lines list."""
```

Reply:
Yeah, we should line wrap those lines that are 80 or more.  I already merged this commit, so feel free to create a new PR to handle above.